lib/src/migrators/module.dart

```diff
@@ -379,9 +378,9 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
         .toSet()) {
       if (url == currentUrl || _forwardedUrls.contains(url)) continue;
       var forwards =
-          _makeForwardRules(url, '"${_absoluteUrlToDependency(url).item1}"');
+          _makeForwardRules(url, '"${_absoluteUrlToDependency(url).$1}"');
       if (forwards == null) continue;
-      var isRelative = _absoluteUrlToDependency(url).item2;
+      var (_, isRelative) = _absoluteUrlToDependency(url);
```

Comment:
Since you're accessing both halves of `_absoluteUrlToDependency()` here, you might as well call it early and assign them both to variables